### PR TITLE
Narrow homepage text under banner

### PR DESCRIPTION
**Pull request by jmille79**
MARBLE-1497 
Screen Shot 2021-03-15 at 2 50 46 PM

### CODE REVIEW

File: src/@ndlib/gatsby-theme-marble/components/Pages/IndexPage/index.js [hunk 1]
```diff
@@ -60,7 +60,7 @@ const IndexPage = ({ location }) => {
   return (
     <React.Fragment>
       <BaseStyles>
-        <p dangerouslySetInnerHTML={{ __html: t('common:hompageDescriptive') }} />
+        <p sx={{ margin: '3rem' }} dangerouslySetInnerHTML={{ __html: t('common:hompageDescriptive') }} />
```
Comment (danthewolfe) [inline src/@ndlib/gatsby-theme-marble/components/Pages/IndexPage/index.js:63]: I think the intention here is to set a maximum text block width for accessibility. So this needs to be a `maxWidth` and it probably needs to be different at the different breakpoints. Maybe something like `maxWidth: ['100%', '80rem', '80rem']` (Spitballing numbers here. I think I remember maximum accessible text width was around 70 or 80 characters wide.)

Thoughts @jonhartzler?

Comment (jonhartzler) [reply src/@ndlib/gatsby-theme-marble/components/Pages/IndexPage/index.js:63]: yes, this should be max-width.  

WCAG standards have a maximum width a block of text should be for readability and if we put blocks of text larger than that one the site we are in violation of the standard.  So we need a max-width value not just a margin

**Review by jonhartzler — CHANGES_REQUESTED** [review]
See other comment in response to Dan

**Review by danthewolfe — APPROVED** [review]
Assuming this looks okay and @jonhartzler approves 👍

**Pull request merged**